File: Mobile/objects/viewmodels/CustomerOrganizations/CustomerOrganizationViewModel.ts
import { BasicOrganizationViewModel } from "../Organizations/BasicOrganizationViewModel";
import { CustomerUserViewModel } from "../Users/customers/CustomerUserViewModel";

export interface CustomerOrganizationViewModel {
    id?: number;
    customerId?: string;
    organizationId?: number;
    customer?: CustomerUserViewModel;
    organization?: BasicOrganizationViewModel;
}

File: BE/Boek.Infrastructure/ViewModels/CustomerOrganizations/CustomerOrganizationViewModel.cs
using Boek.Infrastructure.Attributes;
using Boek.Infrastructure.ViewModels.Organizations;
using Boek.Infrastructure.ViewModels.Users.Customers;

namespace Boek.Infrastructure.ViewModels.CustomerOrganizations
{
    public class CustomerOrganizationViewModel
    {
        [Int]
        public int? Id { get; set; }

        [Guid]
        public Guid? CustomerId { get; set; }

        [Int]
        public int? OrganizationId { get; set; }

        public CustomerUserViewModel Customer { get; set; }

        public BasicOrganizationViewModel Organization { get; set; }
    }
}
